### PR TITLE
[FT] Extended periods of timeout and grace period to six months

### PR DESCRIPTION
**Pull request by loicemeyo**
*Problem*
Currently the user can only set checks with a limited timeout and grace period - each of only of 30 days. This should be extended as some crone jobs can run longer than 30 days.

*Solution*
- Extend maximum periods from one month to six months on the forms file
-Fix tests to accommodate  samples of extended timeout and grace periods
-Change intervals on the modal for the period-slider and grace-slider

*Relevant Pivotal Tracker story*
https://www.pivotaltracker.com/story/show/158532878

*Screenshot to demonstrate solved problem*
image

### CODE REVIEW

**Review by simiyu1 — CHANGES_REQUESTED** [review]
Commendable use of comments and easy to follow code. Good work, however your tests are failing. Here are some things to watch out for..

Please check your test files for the function 'test_hc_duration_work'. Your assert is looking for "6 months" but finds "1 six_months"

Check also the test function 'test_timeout_too_large', ensure that it can be found because it yeilds a 404 error.

Confirm that '-webkit-text-size-adjust' has been correctly named. Is the hyphen part of the name?

File: static/js/checks.js [hunk 1]
```diff
@@ -29,21 +31,22 @@ $(function () {
 
         return result;
     }
-
+    //To change the intervals on the period slider for viewing and choosing timeouts"""
```
Comment (karanjaE) [inline static/js/checks.js]: you don't need the docstring quotes in js files

File: static/js/checks.js [hunk 2]
```diff
@@ -58,21 +61,22 @@ $(function () {
         $("#update-timeout-timeout").val(rounded);
     });
 
-
+    //To change the intervals on the grace slider for viewing and choosing grace periods"""
```
Comment (karanjaE) [inline static/js/checks.js]: here too

**Review by karanjaE — CHANGES_REQUESTED** [review]

**Review by thoth-ky — COMMENTED** [review]
Fix failing tests

**Review by simiyu1 — APPROVED** [review]

**Review by karanjaE — APPROVED** [review]

**Pull request merged**